src/components/Filters/components/TagsWrapper/TagsWrapper.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import {VisuallyHidden} from '../../../VisuallyHidden';
+
+interface Props {
+  children: React.ReactNode;
+  shouldHide: boolean;
```

Comment:
Gonna be real annoying as I was the one who suggested this in the first place:

Could we rename this prop to `hidden` as we tend to use that tense in other places (titleHidden, labelHidden, ViauallyHidden etc)